projects/plugins/jetpack/_inc/lib/class-jetpack-recommendations.php

```diff
@@ -196,26 +196,18 @@ public static function plugin_activated( $plugin ) {
 		$plugin_file = $path_parts ? array_pop( $path_parts ) : $plugin;
 
 		if ( ! in_array( $plugin_file, $plugin_whitelist, true ) ) {
-			$has_anti_spam = is_plugin_active( 'akismet/akismet.php' );
+			$products              = array_column( Jetpack_Plan::get_products(), 'product_slug' );
+			$has_anti_spam_product = count( array_intersect( array( 'jetpack_anti_spam', 'jetpack_anti_spam_monthly' ), $products ) ) > 0;
+			$has_anti_spam         = is_plugin_active( 'akismet/akismet.php' ) || Jetpack_Plan::supports( 'antispam' ) || $has_anti_spam_product;
```

Comment:
Just pointing out that Akismet could be active without having a plan. I don't think that's a blocker.